File: Rated/LONG/FEB21C/PRIGAME_sub_1.py
# for subtask 1
# y == 1
# ?using Goldbach's conjecture

t = int(input())
for _ in range(t):
    x, y = map(int, input().strip().split())
    if x == 1 or x == 2:
        print('Chef')
    else:
        print('Divyam')

# python go brrrrr
# gave TLE; cpp counterpart got accepted for sub 1

File: Rated/LONG/FEB21C/PRIGAME_sub_1.cpp
#include <iostream>
using namespace std;

int main(){
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);
    cout.tie(NULL);

    int t, x, y;
    cin >> t;
    for(int it = 0; it < t; it++) {
        cin >> x >> y;
        if(x==1 || x==2){cout << "Chef\n";}
        else{cout << "Divyam\n";}
    }
    return 0;
}
// for subtask 1
// y == 1
// goldbach conjecture?